mindsdb/integrations/utilities/files/file_reader.py

```diff
@@ -65,6 +66,7 @@ def decode(file_obj: IOBase) -> StringIO:
 class FormatDetector:
 
     supported_formats = ['parquet', 'csv', 'xlsx', 'pdf', 'json', 'txt']
+    multipage_formats = ['xlsx']
```

Comment:
How do multi-page PDFs work here? I see that read_pdf() is extracting data from all pages. We return the data in all files as a single chunk, is it?

Reply:
PDF is parsed as a text, there are not specific column names. For xlsx we have different sheets with different set of columns. By some reason I decide to use multi-page term instead of multi-sheet and not it confuses with pates in formats like pdf, but we can discuss another them

Reply:
Hmm.. Since we are parsing multi-page documents like .pdfs and .txts as a single chunk, I guess only .xlsx (there could be other Excel variants like .xls, I guess) files will be treated this way, but I guess it makes sense to keep a list in case something comes up later. Maybe `multi_part_formats` or `multi_content_formats`?

Reply:
first thought: it is possible to load sqlite db and select tables as pages. 
maybe we can rename it to `multitable_formats`